lib/parallel_tests/cli.rb

```diff
@@ -142,7 +142,19 @@ def report_number_of_tests(groups)
       num_processes = groups.size
       num_tests = groups.map(&:size).inject(0, :+)
       tests_per_process = (num_processes == 0 ? 0 : num_tests / num_processes)
-      puts "#{num_processes} processes for #{num_tests} #{name}s, ~ #{tests_per_process} #{name}s per process"
+      puts "#{pluralize(num_processes, 'process'} for #{pluralize(num_tests, name)}, ~ #{pluralize(tests_per_process, name)} per process"
+    end
+
+    def pluralize(n, singular, plural=nil)
+      if n == 1
+        "1 #{singular}"
+      elsif plural
+        "#{n} #{plural}"
```

Comment:
we never pass plural 🤷 
```suggestion
    def pluralize(n, singular)
      if n == 1
        "1 #{singular}"
```

Reply:
The idea was to include future proofing for weird stuff like person/people. :) If it’s cleaner to remove it, no problem!

Reply:
Okay, pushed with your changes!